extensions/functions_arithmetic.yaml

```diff
@@ -1071,6 +1071,86 @@ aggregate_functions:
             value: fp64
         nullability: DECLARED_OUTPUT
         return: fp64?
+  - name: "quantile"
+    description: >
+      Calculates quantiles for a set of values.
+      
+      This function will divide the aggregated values (passed via the
+      distribution argument) over N equally-sized bins, where N is passed
+      via a constant argument. It will then return the values at the
+      boundaries of these bins in list form. If the input is appropriately
+      sorted, this computes the quantiles of the distribution.
+      
+      The function can optionally return the first and/or last element of
+      the input, as specified by the `boundaries` argument. If the input is
+      appropriately sorted, this will thus be the minimum and/or maximum
+      values of the distribution.
+      
+      When the boundaries do not lie exactly on elements of the incoming
+      distribution, the function will interpolate between the two nearby
+      elements. If the interpolated value cannot be represented exactly,
+      the `rounding` option controls how the value should be selected or
+      computed.
+      
+      The function fails and returns null in the following cases:
+        - `n` is null or less than one;
+        - any value in `distribution` is null.
+      
+      The function returns an empty list if `n` equals 1 and `boundaries` is
+      set to `NEITHER`.
+    
+    impls:
+      - args:
+          - name: boundaries
+            description: >
+              Which boundaries to include. For NEITHER, the output will have
+              n-1 elements, for MINIMUM and MAXIMUM it will have n elements,
+              and for BOTH it will have n+1 elements.
+            options: [ NEITHER, MINIMUM, MAXIMUM, BOTH ]
+            required: true
+          - name: precision
+            description: >
+              Based on required operator performance and configured optimizations
+              on saving memory bandwidth, the precision of the end result can be
+              the highest possible accuracy of an approximation. 
+
+                - EXACT: provides the highest accurate output
+                - APPROXIMATE: provides a sub-optimal output
```

Comment:
> I updated the description of the `precision`.

I mean here. Or maybe you forgot to push?

Reply:
Is it okay now?